P5/Source/Specs/noteGrp.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
+    Copyright TEI Consortium.
+    Dual-licensed under CC-by and BSD2 licences
+    See the file COPYING.txt for details
+    $Date$
+    $Id$
+--><?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?><elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="core" ident="noteGrp">
+  <desc versionDate="2005-01-14" xml:lang="en">contains a group of notes</desc>
+  <classes>
+    <memberOf key="att.global"/>
+    <memberOf key="model.noteLike"/>
+    <memberOf key="att.placement"/>
+    <memberOf key="att.pointing"/>
+    <memberOf key="att.typed"/>
+    <memberOf key="att.written"/>
+    <memberOf key="model.correspDescPart"/>
+    <memberOf key="model.correspActionPart"/>
+    <memberOf key="model.correspContextPart"/>
+  </classes>
+  <content>
+    <sequence>
+        <classRef key="model.headLike" minOccurs="0"/>
```

Comment:
I am thinking that if 1 heading is allowed, more than 1 should be allowed. Thus `<classRef key="model.headLike" minOccurs="0" maxOccurs="unbounded"/>`, no?

Reply:
Thanks Syd for focusing on this! I had a look at other *Grp elements and only `spGrp` allows any headings (but it does allow multiple too). 
I think any headings make sense only for transcribing (e.g. printed) sources, but I believe for such purposes we were planning a `listNote` element and `noteGrp` would be something. So perhaps I'll remove model.headLike from the noteGrp altogether?

Reply:
I think @hcayless’ “don’t put it in unless requested” philosophy would definitely have us leave `<head>` out for now. And while I often think particular cases override that philosophy, I do not think this one does.